httpx/_client.py

```diff
@@ -1130,6 +1180,11 @@ async def request(
         allow_redirects: bool = True,
         timeout: typing.Union[TimeoutTypes, UnsetType] = UNSET,
     ) -> Response:
+        """
+        Build and send a request.
+
+        Merges parameters with the client.
+        """
```

Comment:
```suggestion
        """
        Build and send a request.

        Roughly equivalent to:

        ```python
        request = client.build_request(...)
        response = client.send(request, ...)
        ```

        See `Client.build_request()`, `Client.send()` and [Merging of configuration][0]
        for how the various parameters are merged with client-level configuration.

        [0]: /advanced/#merging-of-configuration
        """
```

Reply:
Added this to the new version!